flink-runtime/src/main/java/org/apache/flink/runtime/state/CheckpointStreamFactory.java

```diff
@@ -43,9 +44,9 @@ CheckpointStateOutputStream createCheckpointStateOutputStream(
 	 * Closes the stream factory, releasing all internal resources, but does not delete any
 	 * persistent checkpoint data.
 	 *
-	 * @throws Exception Exceptions can be forwarded and will be logged by the system
+	 * @throws IOException Exceptions can be forwarded and will be logged by the system
 	 */
-	void close() throws Exception;
+	void close() throws IOException;
```

Comment:
What's this change for? Is it used anywhere? It seems to me like `close()` is just a dead code and this method could be dropped, or did I missed something?

Reply:
I think this method is actually removed in the latest master.